conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -470,38 +437,22 @@ public void testBinaryServerSideFailureAfterFewBytesSent() {
     public void testBinaryServerSideFailureAfterFewBytesReceived() {
         byte[] data = new byte[1024];
         ThreadLocalRandom.current().nextBytes(data);
-        assertThatThrownBy(() -> {
-                    try {
-                        Futures.getUnchecked(binaryClient.postBinaryThrows(
-                                AuthHeader.valueOf("authHeader"), 1024 * 1024, new RequestBody() {
-                                    @Override
-                                    public MediaType contentType() {
-                                        return MediaType.get("application/octet-stream");
-                                    }
-
-                                    @Override
-                                    public void writeTo(BufferedSink sink) throws IOException {
-                                        // 1gb
-                                        for (int i = 0; i < 1024 * 1024; i++) {
-                                            sink.write(data);
-                                        }
-                                    }
-                                }));
-                    } catch (UncheckedExecutionException e) {
-                        throw e.getCause();
-                    }
-                })
+        assertThatThrownBy(() -> binaryClient.postBinaryThrows(
+                        AuthHeader.valueOf("authHeader"),
+                        1024 * 1024,
+                        BinaryRequestBody.of(new ByteArrayInputStream(data))))
```

Comment:
I don't think this is equivalent. SequenceInputStream perhaps?

Reply:
I will restore 👍🏼

Reply:
I am not sure I understand the purpose of this test. You can send it less or more and it always throws the same thing. We aren't testing if it stops consuming before it sends too much data. Basically I cannot make this thing fail to show that I have restored its behavior with my implementation. I checked out develop, twiddled with the numbers, and I could not make it fail. Should I be testing that its severing the stream before all the copies are written to it?